### PR TITLE
Re-style withheld list

### PR DESCRIPTION
**Pull request by shawnbot**
Fixes issue(s) #2023.

[CircleCI](https://circleci.com/gh/18F/doi-extractives-data/tree/withheld-list)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/doi-extractives-data/withheld-list/)

Before:

image

After:

image

Changes proposed in this pull request:
- Re-styles the withheld product list under federal production, e.g. in [Utah](https://federalist.18f.gov/preview/18F/doi-extractives-data/withheld-list/explore/UT/)
  - [x] Turn the section into a chart-like block with similar title
  - [x] Remove bullets
  - [ ] Normalize text size
  - [x] ~~Style the button with the plus-in-circle icon, no background color~~ (no state has more than three withheld products)

@gemfarmer @ericronne: I'm not sure what the text size for this should be, but 0.875rem appears to be the next size down from 1. I'm also not sure what the "right" way is to reduce either the line-height or the amount of whitespace between the list items. 😬

### CODE REVIEW

**Comment by ericronne** [comment]
@shawnbot i'm having trouble finding a page with an example of this; could you link me?
thx!

**Comment by gemfarmer** [comment]
@shawnbot is this still a WIP?

**Comment by ericronne** [comment]
@shawnbot here's the formatting i had. I figured it would only occupy one column, and slot in as the last chart of the group.

image

I had also dropped in the W icon parenthetically, which we should only do if there's time to apply that to charts that have withheld years (issue #2008).

image
image

thx!

**Comment by ericronne** [comment]
@shawnbot is it possible to reduce the width down to match the bar charts, as opposed to running the full width of the page? It's fine if there's neg space to the right.

**Comment by gemfarmer** [comment]
@shawnbot I like the list styling way more! Are we supposed to be seeing the 'W' in the bars? https://cloud.githubusercontent.com/assets/9259185/20358942/6e178d88-abf3-11e6-88b5-d50a8b786cd1.png

**Comment by gemfarmer** [comment]
@shawnbot looks like these three lines are redundant!

**Comment by gemfarmer** [comment]
Well withheld icons or not, looks great! I vote that we merge once the Jekyll error goes away.

**Comment by ericronne** [comment]
What about making it a single column?
(We can deal with #2008 in a sep PR, time permitting.)

File: js/components/open-list-nav.js [hunk 1]
```diff
@@ -126,11 +126,13 @@
 
             if (!document.getElementById(item.dataset.navItem) && item) {
               item.setAttribute('aria-hidden', true);
+              item.outerHTML = '';
+              delete item;
+            } else {
+              item.addEventListener('click', function () {
```
Comment (dmsnell) [inline js/components/open-list-nav.js]: Seriously @shawnbot - why would you do that! Never…ever…make functions within a loop.

:wink: 🙃

**Pull request merged**